src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -55,24 +55,55 @@ def get_cookie_value(self, name):
 
     @keyword
     def get_cookie(self, name):
-        """Returns the cookie found with `name` in extended variable format.
-
-        If no cookie is found with `name`, this keyword fails.
+        """Returns a cookie object found with ``name``.
+
+        If no cookie is found with ``name``, keyword fails. The cookie object
+        contains details about the cookie. Attributes available in the object
+        are documented in the table below.
+        | = Attribute = |             = Explanation =                            |
+        | domain        | Specifies hosts to which the cookie will be sent       |
+        | expiry        | The maximum lifetime of the cookie as EPOCH            |
+        | httpOnly      | HttpOnly cookie cannot be accessed by client-side APIs |
+        | name          | The name of a cookie                                   |
+        | path          | Indicates a URL path, example /                        |
+        | secure        | Cookie will be send only by using secure connection    |
+        | value         | Value of the cookie                                    |
+        | full_info     | All the above attributes joined as string              |
```

Comment:
`__str__` seems to have all the information already. I don't think we need `full_info` at all.

Reply:
I was thinking about that. Having `name` and `value` did feel too little information. But again having all feels too much.

I think most of the people are only interested about the `name` and `value` and not about the other details. Perhaps it would be good idea to revert this back to `name` and `value`.

Reply:
I'd rather remote `full_info` altogether and have all info in `__str__` in format like this:

    name=<name>
    value=<value>
    path=<path>
    ...

Reply:
Done